riff-raff.yaml

```diff
@@ -6,7 +6,7 @@ deployments:
     type: aws-lambda
     parameters:
       bucket: mobile-dist
-      functionNames: [live-app-versions-]
+      functionNames: [live-app-versions-, ios-deployments-]
       fileName: live-app-versions.jar
```

Comment:
We just share the same `.jar` file for now. These lambdas are both going to run on a schedule so I don't think the performance overhead caused by increased `.jar` size is going to warrant a more complex structure in `build.sbt` yet.

Reply:
You might be able to copy paste the function definition with a a new name, deploy, then delete the old definition and deploy again.
Which is the same amount of operations so really it's not simpler :/